src/pinejs-session-store/pinejs-session-store.ts

```diff
@@ -30,25 +30,45 @@ const sessionModel = `
 		Necessity: Each session has at most 1 expiry time
 `;
 
+const asCallback = async <T>(
+	callback: undefined | ((err: any, result?: T) => void),
+	promise: Promise<T>,
```

Comment:
I would prefer having the promise as the first arg, imo follows the `fact => result` pattern and I'm used always seeing callbacks last, but not a bit deal :)